netneurotools/networks.py

```diff
@@ -59,10 +59,22 @@ def func_consensus(data, n_boot=1000, ci=95, seed=None):
 
     # group-average functional connectivity matrix desired instead of bootstrap
     if n_boot == 0 or n_boot is None:
-        corrs = [np.corrcoef(data[..., sub]) for sub in range(data.shape[-1])]
+        if type(data) is list:
```

Comment:
Using `isinstance` instead of `type` is generally preferred in these cases!

```suggestion
        if isinstance(data, list):
```